guides/source/testing.md

```diff
@@ -589,9 +589,9 @@ class BlogFlowTest < ActionDispatch::IntegrationTest
 end
 ```
 
-If you remember from earlier in the "Testing Views" section we covered `assert_select` to query the resulting HTML of a request.
+We will take a look at `assert_select` to query the resulting HTML of a request in the "Testing Views" section below. It is used for testing the response of our request by asserting the presence of key HTML elements and their content. 
```

Comment:
Question: Is this section out of order? We were referring to "Testing Views" which is defined later.
I think, we can more integration tests below others.